build/get_cd_deps.sh

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash
+echo "BUILD_BRANCH: ${BUILD_BRANCH}"
+if [[ $BUILD_BRANCH == "staging" ]]; then
+  echo "Building staging"
+  npm install --save cp-translations@staging @coderdojo/cd-common@staging
+elif [[ $BUILD_BRANCH =~ ^pull\/[0-9]+$ ]]; then
+  echo "Building PR"
+  PR_ID=`echo ${BUILD_BRANCH} | cut -d '/' -f 2`
```

Comment:
we shouls ue `$(echo "${BUILD_BRANCH}" | cut -d '/' -f 2)` rather then ``echo ${BUILD_BRANCH} | cut -d '/' -f 2``